scarb/src/core/package/mod.rs

```diff
@@ -69,8 +68,8 @@ impl Package {
             .ok_or_else(|| anyhow!("package `{self}` has no target `{kind}`"))
     }
 
-    pub fn fetch_profile(&self, profile: &SmolStr) -> Result<Profile> {
-        self.manifest.profiles.fetch_profile(profile)
+    pub fn has_profile(&self, profile: &Profile) -> Result<bool> {
```

Comment:
```suggestion
    pub fn has_profile(&self, profile: &Profile) -> bool {
```

this will never fail I think